src/components/Burst.css

```diff
@@ -1,5 +1,6 @@
 .Burst {
   position: absolute;
+  z-index: 2;
```

Comment:
Again, remove all of these component-specific `z-index` with the exception of Preview, whose changes suggested previously should be kept.